ui/src/components/layouts/HalfMapLayout/NewMap.tsx

```diff
@@ -2,95 +2,201 @@ import "leaflet/dist/leaflet.css";
 
   // TODO: where would it make sense to show all sightings in a longer time range?
   // const { filteredData } = useData();
   // const allSightings = filteredData.filter((el) => {
-  //   return el.newCategory === "SIGHTINGS";
+  //   return el.newCategory === "SIGHTING";
   // });
 
-  const [sightings, setSightings] = useState<CombinedData[]>();
-  useEffect(() => {
-    const sightingsNow = nowPlayingCandidate?.array?.filter((el) => {
-      return el.newCategory === "SIGHTINGS";
-    });
-    setSightings(sightingsNow);
-  }, [nowPlayingCandidate]);
+  const [map, setMap] = useState<LeafletMap>();
+  const [latLng, setLatLng] = useState<LatLngExpression>([48.1, -122.75]);
+  const [zoom, setZoom] = useState<number>(9);
 
   const hydrophoneDefaultIcon = L.icon({
     iconUrl: hydrophoneDefaultIconImage.src,
     iconSize: [30, 30],
   });
+
   const hydrophoneActiveIcon = L.icon({
     iconUrl: hydrophoneActiveIconImage.src,
     iconSize: [30, 30],
   });
 
+  useEffect(() => {
+    if (feed) {
+      setLatLng(feed.latLng);
+      setZoom(12);
+      // map?.setZoom(12);
+      // map?.panTo(feed.latLng);
+    } else {
+      setZoom(9);
+      // map?.setZoom(8);
+    }
+  }, [map, feed, setLatLng, setZoom]);
+
   return (
-    <MapContainer
-      // center={[48.1, -122.75]} // this was formerly the center for the zoomed out but it overrides the first nowPlaying panTo in HalfMapLayout
-      zoom={9}
-      maxZoom={13}
-      style={{ height: "100%", width: "100%" }}
-      ref={setMap}
-      zoomControl={false}
-      //TODO: Disable attribution on mobile only
-      attributionControl={false}
-    >
-      <ZoomControl position="topright" />
-      <TileLayer
-        attribution="Tiles &copy; Esri &mdash; Sources: GEBCO, NOAA, CHS, OSU, UNH, CSUMB, National Geographic, DeLorme, NAVTEQ, and Esri"
-        url="https://server.arcgisonline.com/ArcGIS/rest/services/Ocean/World_Ocean_Base/MapServer/tile/{z}/{y}/{x}"
-      />
-      <TileLayer url="https://server.arcgisonline.com/ArcGIS/rest/services/Ocean/World_Ocean_Reference/MapServer/tile/{z}/{y}/{x}" />
-      {feeds?.map((feed) => (
-        <Marker
-          key={feed.slug}
-          position={feed.latLng}
-          icon={
-            feed.slug === currentFeed?.slug
-              ? hydrophoneActiveIcon
-              : hydrophoneDefaultIcon
+    <>
+      <LeafletTooltipGlobalStyles />
+      <MapContainer
+        center={feed ? feed.latLng : [48.1, -122.75]} // this needs to be set or the map won't initialize
+        zoom={feed ? 12 : 9}
+        maxZoom={13}
+        className="map-container"
+        style={{ height: "100%", width: "100%" }}
+        ref={(instance) => {
+          if (instance) {
+            setMap(instance);
           }
-          eventHandlers={{
-            click: () => {
-              router.push(`/listen/${feed.slug}`);
-            },
-          }}
+        }}
+        zoomControl={false}
+        //TODO: Disable attribution on mobile only
```

Comment:
Update comment to explain why attribution would be disabled on mobile (it's non-obvious to me)